hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.tsx

```diff
@@ -465,6 +467,18 @@ export const PendingTable = () => {
                                 </TableBody>
                             </Table>
                         </TableContainer>
+                        <div className={styles.title}>
+                            <Typography
```

Comment:
Why do you have an empty Typography component?
if you want to use styles.title you can put it directly on the Button component, no need for an extra div wrapper

Reply:
for whatever reason, the button gets rendered to the right underneath the order table when I have the empty wrapper. Otherwise with that class div, its rendered to the left, not sure how to fix it without adding my own css

Reply:
Not sure how to resolve this formatting with CSS... 
![image](https://user-images.githubusercontent.com/43456930/174139637-24a09352-189c-4343-a1b6-87fb5276e8b1.png)